source/messaging/customizing-theme-colors.rst

```diff
@@ -3,7 +3,7 @@
 Theme Colors
 ============
 
-The colors of the Mattermost user interface are customizable in **Account Settings > Display > Theme**. You can import your theme colors from Slack, customize the colors yourself, or choose from four standard themes designed by the Mattermost team.
+The colors of the Mattermost user interface are customizable in **Account Settings > Display > Theme**. You can import your theme colors from Slack, customize the colors yourself, or choose from four standard themes designed by the Mattermost team - "Denim", "Sapphire", "Indigo", and "Onyx".
```

Comment:
missing quartz

Reply:
Thanks! I updated the release notes as well. 
```suggestion
The colors of the Mattermost user interface are customizable in **Account Settings > Display > Theme**. You can import your theme colors from Slack, customize the colors yourself, or choose from four standard themes designed by the Mattermost team - "Denim", "Sapphire", "Quartz", "Indigo", and "Onyx".
```